### ISSUE TITLE
Deleting an instance doesn't always focus the next selected one

### ISSUE THREAD

**Issue by johnsicili** [webstudio-is/webstudio#4044]
When I delete an instance, the next in line is selected, but it doesn't have the focus outline.

The issue is rooted in nested instances. Top-level ones don't lose focus.

1. Add multiple instances nested
2. Delete one
3. See the next has blue bg but no outline

https://github.com/user-attachments/assets/dd8b4e94-7950-4260-9866-301531f60962

**Comment by johnsicili** [comment]
Doesn't appear to be an issue anymore. Maybe new navigator fixed it.
